quic/s2n-quic-transport/src/connection/connection_container.rs

```diff
@@ -940,6 +955,51 @@ impl<C: connection::Trait, L: connection::Lock<C>> ConnectionContainer<C, L> {
         );
     }
 
+    /// Iterates over all `Connection`s which are waiting to process ACKs,
+    /// and executes the given function on each `Connection`
+    pub fn iterate_ack_list<F>(&mut self, mut func: F)
```

Comment:
I think we can still use the iterate_interruptible macro but just wrap the function in one that always continues:

```
        iterate_interruptible!(
            self,
            waiting_for_connection_id,
            waiting_for_connection_id_link,
            |conn| {
                func(conn);
                ConnectionContainerIterationResult::Continue
            }
        );
```

Reply:
implemented in https://github.com/aws/s2n-quic/pull/1278/commits/40cbae3e4431d6adab9281d4527a6b8143d9082c. Note that i had to change `ident` to `tt`. I was getting borrow issues (`connection.inner.write(|conn|`) with the closure otherwise.

Reply:
thats fine. although it's _probably_ better as a `expr` rather than `tt`. but not a huge deal.